tensorflow_probability/python/mcmc/transformed_kernel.py

```diff
@@ -33,15 +33,11 @@
     'TransformedTransitionKernel',
 ]
 
-
-class TransformedTransitionKernelResults(
-    mcmc_util.PrettyNamedTupleMixin,
-    collections.namedtuple('TransformedTransitionKernelResults',
-                           ['transformed_state',
-                            'inner_results',
-                            ])):
-  """Internal state and diagnostics for Transformed kernel."""
-  __slots__ = ()
+TransformedTransitionKernelResults = collections.namedtuple(
+    'TransformedTransitionKernelResults', [
+        'transformed_state',
```

Comment:
looks like some recent changes to this file have been lost here

Reply:
Yup, fixed